app.rb

```diff
@@ -0,0 +1,164 @@
+# rubocop:disable Metrics/MethodLength, Lint/MissingCopEnableDirective
+# rubocop:disable Metrics/CyclomaticComplexity
+require_relative 'book'
+require_relative 'person'
+require_relative 'students'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rental = []
+  end
+
+  # Create menu that allows to do his choice
+  def menu_showing
+    puts "\n"
+    puts 'Press 1 to show the list of books'
+    puts 'Press 2 to show the list of people'
+    puts 'Press 3 to add a person'
+    puts 'Press 4 to add a book'
+    puts 'Press 5 to add a rental'
+    puts 'Press 6 to show the list of rentals'
+    puts 'Press 7 to quit'
+    puts "\n*********************************************************\n"
+    print 'Enter your choice: '
+    gets.chomp.to_i
+  end
+
+  # Create actions for each menu choice
+  def menu_choice(action = menu_showing)
+    case action
+    when 1
+      show_books
+    when 2
+      show_people
```

Comment:
When I select option 2 to list all people. The application quits.

![image](https://user-images.githubusercontent.com/38088818/181794183-a73d9274-6442-422b-b44e-58fbb06bd694.png)

- Please make sure that after listing people there's a way I can continue using the application without quitting. This is more convenient because after the application quits all the data is lost, therefore, when I run the application again I have to create all the data from scratch. This is very time-consuming. You can replicate what you did for the successful book creation where I am redirected to the main menu after a book is created.
- Kindly make sure that the application only quits when option 7 is chosen, otherwise, the application should be a continuous loop up until a user decides to quit.

Kindly do the same thing for listing all books.

![image](https://user-images.githubusercontent.com/38088818/181795627-f1dcb715-1ad3-4ad4-ae5e-b53561d27ea7.png)